packages/components/src/index.ts

```diff
@@ -13,6 +13,7 @@ export * from './Select';
 export * from './spinners/Spinner';
 export * from './spinners/CenteredSpinner';
 export * from './spinners/SpinnerOverlay';
+export { sideOptions } from './internal/popup-utils';
```

Comment:
@cahrens , I took Nick's advice (thanks @nawatts) and defined this `sideOptions` array, but it doesn't seem right to be exporting something out of the `internal` directory. Any thought about where this should go instead? or am I overthinking it?

Reply:
thanks Christina -- I'm mostly thinking about following conventions given the directory name (`internal`). But maybe I am just overthinking it.

Reply:
Ah, I see. Well, you could always re-export it from a different file like `PopupTrigger` (which would import it from the internal one), but I don't personally think it's a big deal.